src/analyzer/protocol/dns/DNS.cc

```diff
@@ -196,9 +196,16 @@ bool DNS_Interpreter::ParseQuestion(DNS_MsgInfo* msg,
 
 	if ( dns_event && ! msg->skip_event )
 		{
-		BroString* question_name =
-			new BroString(name, name_end - name, true);
-		SendReplyOrRejectEvent(msg, dns_event, data, len, question_name);
+	    BroString* original_name = new BroString(name, name_end - name, true);
```

Comment:
The spaces here should be tabs

Reply:
https://github.com/zeek/zeek/pull/1015/commits/7a91b49f5a25ce91575383cd45cdd4bba971515a